Content.Client/MassMedia/Ui/ArticleEditorPanel.xaml.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Content.Client.Message;
+using Content.Client.Resources;
+using Content.Client.Stylesheets;
+using Robust.Client.AutoGenerated;
+using Robust.Client.ResourceManagement;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Animations;
+using Robust.Shared.Utility;
+
+namespace Content.Client.MassMedia.Ui;
+
+[GenerateTypedNameReferences]
+public sealed partial class ArticleEditorPanel : Control
+{
+    public event Action? PublishButtonPressed;
+
+    private bool _preview = false;
+
+    public ArticleEditorPanel()
+    {
+        RobustXamlLoader.Load(this);
+
+        ButtonPublish.StyleClasses.Add(StyleBase.ButtonOpenLeft);
+        ButtonPublish.StyleClasses.Add(StyleNano.StyleClassButtonColorGreen);
+
+        ContentField.GetChild(0).Margin = new Thickness(9, 3);
+        var scrollbar = ContentField.GetChild(1);
+        scrollbar.SetWidth = 6f;
+        scrollbar.Margin = new Thickness(9, 0, 2 , 0);
```

Comment:
is there a way to move margins value to the xaml to avoid hardcoding?

Reply:
duplicate